compiler/rustc_middle/src/ty/fold.rs

```diff
@@ -796,14 +795,14 @@ impl<'tcx> TyCtxt<'tcx> {
         mut fld_r: F,
         fld_t: G,
         fld_c: H,
-    ) -> (T, BTreeMap<ty::BoundRegion, ty::Region<'tcx>>)
+    ) -> (T, FxHashMap<ty::BoundRegion, ty::Region<'tcx>>)
```

Comment:
Does any use site rely on the iteration order?

Reply:
Maybe here?  https://github.com/rust-lang/rust/blob/181e91567c9f347e055b33b1d7e9894f769aafe3/compiler/rustc_borrowck/src/type_check/mod.rs#L1589